core/src/main/resources/lib/layout/header/searchbox.jelly

```diff
@@ -1,25 +1,12 @@
 <?jelly escape-by-default='true'?>
-<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:l="/lib/layout">
-  <div class="searchbox hidden-xs">
-    <!-- search box -->
-    <j:set var="searchURL" value="${h.searchURL}"/>
-    <form action="${searchURL}" method="get" style="position:relative;" class="no-json" name="search" role="search">
-      <!-- this div is used to calculate the width of the text box -->
-      <div id="search-box-sizer"/>
-      <div id="searchform">
-        <input name="q" placeholder="${searchPlaceholder}" id="search-box" class="main-search__input" value="${request.getParameter('q')}" role="searchbox" />
-
-        <span class="main-search__icon-leading">
-          <l:icon src="symbol-search"/>
-        </span>
-        <a href="${searchHelpUrl}" class="main-search__icon-trailing">
-          <l:icon src="symbol-help-circle"/>
-        </a>
-
-        <div id="search-box-completion" data-search-url="${searchURL}" />
-        <st:adjunct includes="jenkins.views.JenkinsHeader.search-box" />
```

Comment:
What does this mean for custom header compatibility?

Reply:
once https://github.com/jenkinsci/customizable-header-plugin/pull/101 is merged there is no impact

Reply:
cc @mawinter69

Reply:
I think the search-box.js can then also be removed and the `createSearchBox` method in hudson-behaviour

Reply:
done in https://github.com/jenkinsci/jenkins/pull/7569/commits/f4f567aee4ae6c7cc64d14342affa44e55ef22d8